src/views/admin/PendingCredits.vue

```diff
@@ -7,9 +7,10 @@
             class="accounts card custom-card custom-card-padding"
           >
             <h2 class="custom-card-title">Opérations de crédit en attente de validation</h2>
+            <p class="notification is-danger is-light" v-if="hasLoadingError">Liste des demandes de crédit indisponible</p>
```

Comment:
same comment about being explicit and give an action to move forward

Reply:
Yes, will do.

Reply:
Changed.